webapp/portlet/src/main/webapp/dlp-quarantine/components/DlpQuarantineApp.vue

```diff
@@ -53,7 +53,8 @@
           <v-btn
             v-exo-tooltip.bottom.body="$t('documents.dlp.quarantine.previewDownload')"
             icon
-            text>
+            text
+            @click.prevent="openPreview(item.refernece, item.title, item.downloadUrl)">
```

Comment:
can you check that there is no miss-spelled on item.reference ?

Reply:
Done !